internal/app/machined/pkg/runtime/v1alpha1/v1alpha1_sequencer_tasks.go

```diff
@@ -89,7 +89,7 @@ func SetupLogger(seq runtime.Sequence, data interface{}) (runtime.TaskExecutionF
 		// disable ratelimiting for kmsg, otherwise logs might be not visible.
 		// this should be set via kernel arg, but in case it's not set, try to force it.
 		if err = krnl.WriteParam(&kernel.Param{
-			Key:   "kernel.printk_devkmsg",
+			Key:   "proc.sys.kernel.printk_devkmsg",
```

Comment:
does this mean a machine patch like:

```yaml
- op: add
  path: /machine/sysctls
  value:
    fs.inotify.max_user_instances: "512"
    fs.inotify.max_user_watches: "524288"
```

needs to be updated as:

```yaml
- op: add
  path: /machine/sysctls
  value:
    proc.sys.fs.inotify.max_user_instances: "512"
    proc.sys.fs.inotify.max_user_watches: "524288"
```

Reply:
my understanding is that no, it doesn't need to be updated.

translation happens in a controller where the `sysctls` and `sysfs` are translated into proper `KernelParamSpec` resources